packages/ubuntu_bootstrap/integration_test/ubuntu_bootstrap_test.dart

```diff
@@ -653,6 +728,12 @@ Future<void> verifySubiquityConfig({
         expect(actualStorage.where((config) => config['type'] == 'zpool'),
             isNotEmpty);
         break;
+      case GuidedCapability.ZFS_LUKS_KEYSTORE:
+        expect(actualStorage.where((config) => config['type'] == 'zpool'),
+            isNotEmpty);
+        expect(actualStorage.where((config) => config['type'] == 'dm_crypt'),
+            isNotEmpty);
+        break;
```

Comment:
@spydon sorry, didn't notice you had already approved this. Added a proper assertion for the integration test here that verifies the cloud-config for encrypted ZFS.